package.json

```diff
@@ -4,6 +4,37 @@
   "description": "Reactive Extensions for modern JavaScript",
   "main": "index.js",
   "sideEffects": false,
+  "exports": {
+    ".": {
+      "node": "./index.js",
+      "default": "./_esm5/index.js"
```

Comment:
can you please also include `es2015` version here and in all the other entry-points?
```suggestion
      "default": "./_esm5/index.js",
      "es2015": "./_esm2015/index.js"
```

Reply:
we should also forward-port this change to rxjs@7, @crisbeto could you please send a separate PR for that? thank you

Reply:
This change would fix https://github.com/ReactiveX/rxjs/issues/6321 as well.

Reply:
Done. Will send a PR for the v7 branch ASAP.

Reply:
https://github.com/ReactiveX/rxjs/pull/6614